cmflib/metadata_helper.py

```diff
@@ -123,6 +121,15 @@ def get_or_create_context_type(store, type_name, properties: dict = None) -> met
         context_type.id = store.put_context_type(context_type)  # Returns ID
         return context_type
 
+def update_context_custom_properties(store, context_id, context_name: str, properties: dict, custom_properties: dict) -> metadata_store_pb2.Context:
```

Comment:
Have we checked this workflow ??

Reply:
tested multiple times it updates custom properties